### ISSUE TITLE
Update Life Flask Recovery duration for 3.16 values

### ISSUE THREAD

**Issue by Nemykal** [PathOfBuildingCommunity/PathOfBuilding#3475]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

Life Flasks should show 3.16 values for duration (same recovery over half the duration)

### What is the actual behaviour/value?

Currently it shows old (3.15) values for duration

Life Flasks should show new 3.16 values for duration as per [Patch Notes](https://www.pathofexile.com/forum/view-thread/3186390#flasks):

```
Problem:
Instant-recovery Life Flasks are by far the most reliable tool for survivability, 
as the total Life recovery from a Flask is reasonable but spread over a long period of time.

Solution:
Keep the total Life Recovery from all Life Flasks the same, 
but halve the amount of time for this to occur, 
which doubles their healing per second. 
This is a huge improvement to how effective 
non-instant Life Flasks are at keeping you alive in tough situations.
```

### How to reproduce the issue

From new empty build
Items > Craft Item > Magic > Flask: Life > Divine Life Flask (any life flask will do)

Currently it says "Recovers 2400 Life over 7 Second"

In 3.16 it is expected to say "Recovers 2400 Life over 3.5 Seconds" if we can trust [Patch Notes](https://www.pathofexile.com/forum/view-thread/3186390#flasks)

### Build code

```shell
eNq9Wlt32kYQfk5_hQ7vMazuyoH2YBs7nGMcapykfcpZSwuoXrRUWuGQX9_ZlTCKA2a0aZsHwuWb--zsN0r6v31dcWvD8iIV2aBDznodi2WxSNJsMeh8vL96G3Z--_WX_pTK5Yf5eZly9cuvv7zp6_cWZxvGQa5jSZovmPy00-R8AU1rmsklE9mE_iXya5EMOrciYx3rgWZJKnefYk6L4pau2KAzi0G4Y9EiZllysf--Aq5oms1E_MjkdS7KtTa7SdnTRCSAGU-mH-7uG0bTrGkUfH7Tn3K6ZflMUmkV8DLoDCF0umCXdAWvoI3ykqk0hHboBcStXjvdw8KzNWNJQ8j-_4DTnI3mcxbLdMMu8lReLGkW7wPoHZNri52UXKZrnrK84ZV3TOL9D8oj_xj2XkjKL6ezZ6h9RiJ3n_b_Rk7I03F_TuXynEMhDKwo2fEiSyUzFJ6KtBDZT8TXFD1eWrF6SDOjCHeiw82ireiEZvRCFIgS3KRzhkOOZjjcHZxvHFJ5OWU5TCCJd7aVwEzuT5N93GP2dxN4FHfJvuK0NYFHceNM4rQ1ga_4thFSj3RMEp9Rfu-VARm_U9hxFuOUfsxyVrB805irPlKgLup-8vVeFbxjC4aM9IaxeHkN19kdlQzXks8o7_XkKCwqOQp4IDkeUqBFcpTgi-ScRa-BW6bnM80TxKTIWL7YzpYp4wi0zmVTBJXTpgCyHZoiLeMebWjRPFzEfz2aCo5rDgYXIAgk7MWF7v17EtNc_KUYDG8nNsxXosyRFazAqJCny22RxnCHai54x5Iyxo2uZyI2ERu2gjOhWRyQ0n1hjomec6CzWD4GajlvJTGUksaPlyJZsFZG2kvMyvUaJoPqL6zcVZpDlou0cWO-9RHoD0DZL-gaQ3LUIcQa2KPRBm7SxVJmsA_hrbwQwceypKJoEcwejjZxVXKOIo-zx_R7pKQPnL2zel9J0EtokIRHj5h4AmeWalMsEAeygZ7QPXdxjkaQs-zbFq3_OzjKwChLylx1N9rGS4kfzfS7epNW78artcil_vKC8rjQKsfZupRWptfgQmX-S1auHtQ-Vv29ny5N5Cot4i8P5XyuduMOOJPrhX50dTW6uB9_GtUiM6ZnnBULzum6YLAsZynvWGlSl3kGUz2WCDSsfvUSfRoL93Wcp5C-01C1sCFc1WvzaZzaaBEwqAPlDJui--2aqYoXuCzV4_E0Vq-gCFy1MWLC0nspJuksplsE7pmuILAjzoYpV7cjJk8TaN_qWsWg4ZrL04dSojpKLxoIDxTlRoTVJJWozsfAaoqDMF8zwNPI-u7AtB0QDGSJLtmcQTOjDrIeDvf0kWX12NuNuL4-RYVVwOy7ZqvifAt3y5VK6IvnFN8B4NbJS5hsCZvTkqvvfy8pT-VWDePGtzfVA0rtR7EUT4qoVHrUuS0g1zc31S9DLmsVysigM6e8qDqlf58zZlHN8RSrVCO3Chg-NB9WjhNtfUULyfJt3cCDDoxqmMLwxgtDx-lYEtQ1HpQCba-egVbiSvOb_iiBs5rcKrlu9c3Huxv95s1SynXxrtt9eno6W1O5FHP2FUj0WSxW3TWoAS_f6pvirTLUHcKfc_UyHAy0ou5OU796mFpUautP2lflCfFtEkLxJFtVjnWP4AiBnfE0zieh42Jwrhc6CJxtRxFGnxv6QYTR5xCCgEVuD5MVx_dR0Tqui4uWRCFGn-1GmCgcJ_AwZh3i49zzA9tD4DyPRJhmcaNeiMERBzgnAhdEPsqsH4Y2CgdxoNLsO5hyuD2310OVo4cxS2yfOKgwnAhVXegWH1PdyAsxXUDcCGU3CqIA417oBhizdhQQG9XMJPJRE83peSicHYaoiRa4qNniRGGEsuv7mDhIYBPchETPoAB3I7gouy70M6ZdPAJDEhNvz3MwfUBC10G1PTgYoIaaH-JurMBHxWETHzVdCCjE4Fzio_rKdlDl7QWOh7oqI9SQ9COCmpF2cKhb-t0G24EPQN00DVXcTr25FbA_qN_Ul7sPmvl9StmTVTBYWpczmSs6902I1Z-a6ini-J5ROaHrmmSq32rS6dSUE5bDyxTIYK7J8o65KuAfO__6Y3C3qDmmej9jUv_7dlkwoNUiSz4zuhaZ_lrZqTb-CngUpJk3qRnljIvd04EKZpHZE11bw4dtUVBu1Wk8ULFDoi_FbJzY66hzIWRh6NChWE44dc64bClyBTz90XJQLtmHXHLM0otLgv0viblmTvpm1nzzbJr004kKvGd8xWRbQwc6yTexc6L9rrnYqMeO7RU75qeImNhzzXqBmHWee7o8mMS2VXuovQwDR80dz8AfgxnonDKTbK3qEVXL8h3OslF_eSZCqCOJKYRRnJ5xWm2DuvtmJ8kxa1-nfZe55mPfoKeJcfIdTEO4GJCB2z9BfpCVNK-Cb9CV5mX4iXbBjmQlizw1eKhB5u7SbIFjAq0HxdH0Go4LfD1NwnEQeSLmZXCNs-WZRIM5L-ZNfsKl4arkB-62frfeIfUurJdQ_c_PIpunC0D0uy__5_c_lZrmxw==
```

**Comment by Nemykal** [comment]
looks like it's all in `src/Data/Bases/flask.lua`  - guess we just wait for patch to come out?

**Comment by LocalIdentity** [comment]
Fixed in 2.11.0
